lib/response.js

```diff
@@ -34,6 +34,7 @@ var extname = path.extname;
 var mime = send.mime;
 var resolve = path.resolve;
 var vary = require('vary');
+var deprecate = require('depd')('express');
```

Comment:
This line is now duplicated from above on line 18.

Reply:
removed